code/src/index.css

```diff
@@ -11,3 +11,106 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.main {
+  font-family: "Montserrat", sans-serif;
+  height: 100%;
+  background: #b65fdf;
+  background: -webkit-linear-gradient(top right, #b65fdf, #d88f80);
+  background: -moz-linear-gradient(top right, #b65fdf, #d88f80);
+  background: linear-gradient(to bottom left, #b65fdf, #d88f80);
+}
+
+header {
+  padding: 50px;
+}
+
+h1 {
+  text-align: center;
+  margin: 0;
+}
+
+.loading {
+  text-align: center;
+}
+
+.new-thought-container {
+  background: #d9dce8;
+  background: -webkit-linear-gradient(top left, #d9dce8, #f0bdee);
+  background: -moz-linear-gradient(top left, #d9dce8, #f0bdee);
+  background: linear-gradient(to bottom right, #d9dce8, #f0bdee);
+}
+
+.new-thought-form {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 30px;
+}
+
+.new-thought-input {
+  height: 100px;
+}
+
+.submit-btn {
+  width: 200px;
+  height: 40px;
+}
+
+.thoughts-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 30px;
+  margin-top: 30px;
+}
+
+.thought-box,
+.new-thought-container {
+  width: 315px;
+  margin: auto;
+  padding: 20px;
+  border: 1px solid black;
+  -webkit-box-shadow: 6px 5px 2px 1px rgba(40, 40, 40, 0.71);
+  box-shadow: 6px 5px 2px 1px rgba(40, 40, 40, 0.71);
+}
+
+.thought-box {
+  background: #d9e1e8;
+  background: -webkit-linear-gradient(top left, #d9e1e8, #f0bdc8);
+  background: -moz-linear-gradient(top left, #d9e1e8, #f0bdc8);
+  background: linear-gradient(to bottom right, #d9e1e8, #f0bdc8);
```

Comment:
Come on! How beautiful is that gradient?! Looks warm yet clean! Also great that you take different browsers in consideration! I need to think about this more in my own projects, I often forget it..